src/main.cpp

```diff
@@ -1025,6 +1025,13 @@ bool MainWindow::LoadProject( const wxString &file )
 		VersionUpgrade upgd;
 		upgd.Run( pf );
 		upgd.ShowReportDialog( file );
+        std::vector<Case*> cases = m_project.GetCases();
+		if (m_project.GetCases().size() == 0) {
+			if (wxMessageBox("Load project file with no cases?", "Query", wxYES_NO | wxICON_EXCLAMATION) == wxNO) {
```

Comment:
I got this prompt for a file with two retired technologies and 1 technology that is not retired. A Yes response still gives the unretired case but the prompt is misleading in this case because there are still valid cases in the file.